src/app/contexts/ToggleContext/utils/constructTogglesEndpoint/index.js

```diff
@@ -1,6 +1,6 @@
 export default (service, origin) => {
   const requestOrigin = origin || 'https://www.test.bbc.com';
-  const baseTogglesUrl = `${process.env.SIMORGH_TOGGLES_URL}?application=simorgh&service=${service}&__amp_source_origin=${requestOrigin}`; // __amp_source_origin is relevant to both canonical and amp
+  const baseTogglesUrl = `${process.env.SIMORGH_CONFIG_URL}?application=simorgh&service=${service}&__amp_source_origin=${requestOrigin}`; // __amp_source_origin is relevant to both canonical and amp
```

Comment:
```suggestion
  const baseConfigUrl = `${process.env.SIMORGH_CONFIG_URL}?application=simorgh&service=${service}&__amp_source_origin=${requestOrigin}`; // __amp_source_origin is relevant to both canonical and amp
```

Reply:
I'm not sure about renaming these to `config` as this will imply also changing the file name to `constructConfigEndpoint` and also changing the entire `ToggleContext` to `ConfigContext`. I think semantically what we want is the toggles. Hence, the naming here is fine as `toggles` IMO